tests/Responses/Expected/channel/create_validation_fail_response.json

```diff
@@ -43,7 +43,8 @@
         "errors": [
           "Please enter channel base currency."
         ]
-      }
+      },
+      "shippableCountries": {}
```

Comment:
Maybe we could modify Channels API also, to provide shippable countries (and of course test it properly)?